taskfile/node_git.go

```diff
@@ -22,10 +21,11 @@ import (
 // An GitNode is a node that reads a Taskfile from a remote location via Git.
 type GitNode struct {
 	*BaseNode
-	URL    *url.URL
```

Comment:
I saw no point in keeping this full URL type since it was only used to retrieve the base URL.